playbooks/georep-init-slave-pem-setup.yml

```diff
@@ -4,6 +4,10 @@
   gather_facts: no
 
   tasks:
+  - name: Enable shared storage for geo-rep slave volume
```

Comment:
Shared volume not required in Slave

Reply:
@aravindavk I followed the document available at https://access.redhat.com/documentation/en-us/red_hat_gluster_storage/3.4/html-single/administration_guide/index#idm139715244228960

10.3.4 point number 3:
Enable shared storage for master and slave volumes

Should I raise a documentation bug to change that?

Reply:
@kotreshhr do you know any reason to enable in Slave side?

Reply:
I guess its due to snap scheduler.